questions.md

```diff
@@ -1 +1,3 @@
 Why would you use pull requests instead of just merging branches?
```

Comment:
Pull requests are for code review collaboration purposes. You may want to share your work with your team before you merge it to master :) Hope this answers your question.